src/packer.pkr.hcl

```
@@ -40,6 +40,12 @@ variable "release_url" {
   type        = string
 }
 
+variable "packages_dir" {
+  default     = "packages"
+  description = "The directory where the list of packages are stored."
```

Since I see two package lists I think this reads correctly.

```suggestion
  description = "The directory where the lists of packages are stored."
```